### PR TITLE
fix typing errors

### PR DESCRIPTION
**Pull request by DanielMaass**

### CODE REVIEW

File: dapps/bcc/dbcp.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
       "type": "library"
     },
     "dbcpVersion": 2,
-    "description": "Contractus for loading ens entries and it's data...",
+    "description": "Contractus for loading ens entries and it's data",
```
Comment (karladler) [inline dapps/bcc/dbcp.json]: is by `Contractus` the old company name of evan meant or `Contracts` ? 

```suggestion
    "description": "Contracts for loading ens entries and it's data",
```

Comment (datrinh) [reply dapps/bcc/dbcp.json]:
```suggestion
    "description": "Contracts for loading ens entries and their data",
```

Comment (datrinh) [reply dapps/bcc/dbcp.json]: Almost @karladler :p

**Review by datrinh — CHANGES_REQUESTED** [review]
See comments

**Review by Tschuck — APPROVED** [review]

**Pull request merged**